lib/msal-browser/src/controllers/NestedAppAuthController.ts

```diff
@@ -169,7 +222,10 @@ export class NestedAppAuthController implements IController {
                     reqTimestamp
                 );
 
-            this.operatingContext.setActiveAccount(result.account);
+            // cache the tokens in the response
+            await this.hydrateCache(result, request);
+
+            this.browserStorage.setActiveAccount(result.account);
```

Comment:
Okay either way. Because we cannot guarantee the NAA app's lifetime, it is needed to check the `activeAccount` matches the `accountContext` always, so removing this does not make a difference, I guess. Will make the changes.